html/dialog/demos/polyfilled/index.html

```diff
@@ -0,0 +1,181 @@
+      line-height: 1.5;
+      -webkit-font-smoothing: antialiased;
+      font-family: Roboto, sans-serif;
+    }
+
+    @media (min-width: 580px) {
+      body {
+        align-content: center;
+        align-items: center;
+      }
+    }
+
+    .button {
+      --back: #ffd829;
+      width: 100%;
+      padding: 10px 15px 12px;
+      text-align: center;
+      border: 2px solid transparent;
+      border-radius: 6px;
+      color: #000000;
+      font-family: inherit;
+      font-weight: 300;
+      font-size: 1rem;
+      transition: background-color 0.2s linear;
+      cursor: pointer;
+      background-color: var(--back);
+      outline: none;
+      position: relative;
+    }
+
+    .button:hover {
+      background-color: #ffffff;
+      transition: background-color 0.2s linear;
+    }
+
+    .button:focus {
+      border-color: #ffffff;
+    }
+
+    .button--secondary {
+      --back: #41e847;
+    }
+
+    .button--dark {
+      --back: #000000;
+      color: #ffffff;
+    }
+
+    .button--dark:hover {
+      background-color: rgba(0, 0, 0, 0.6);
+    }
+
+    .button--dark:focus {
+      border-color: #ffd829;
+    }
+
+    .button:focus-visible {
+      outline: 2px solid var(--back);
+      outline-offset: 2px;
+      border-color: transparent;
+    }
+
+    .dialog {
+      width: calc(100% - 40px);
+      max-width: 400px;
+      border: none;
+      inset: 0;
+      transition: transform 0.3s ease-in-out;
+      position: fixed;
+    }
+
+    .dialog::-webkit-backdrop {
+      background-color: rgba(0, 0, 0, 0.6);
+      -webkit-backdrop-filter: blur(10px);
+      backdrop-filter: blur(10px);
+    }
+
+    .dialog::backdrop {
+      background-color: rgba(0, 0, 0, 0.6);
+      -webkit-backdrop-filter: blur(10px);
+      backdrop-filter: blur(10px);
+    }
+
+    .dialog+.backdrop {
+      background-color: rgba(0, 0, 0, 0.6);
+      -webkit-backdrop-filter: blur(10px);
+      backdrop-filter: blur(10px);
+    }
+
+    .dialog__title {
+      font-size: 26px;
+      font-weight: 500;
+      margin: 0;
+    }
+
+    .dialog p {
+      margin: 0.2em 0 1.5em;
+      font-size: 16px;
+    }
+
+    @media (max-width: 579px) {
+      .dialog {
+        /* Сдвигаем диалоговые окна ближе к низу страницы, чтобы они не перекрывали кнопки на демке */
+        margin-bottom: 30px;
+        bottom: 0;
+      }
+    }
+
+  </style>
+</head>
+
+<body>
+  <button class="button button--secondary" type="button" onclick="window['some-dialog-id'].show();"
+    aria-controls="some-dialog-id">Открыть диалоговое окно</button>
+  <button class="button" type="button" onclick="window['some-modal-id'].showModal();"
+    aria-controls="some-modal-id">Открыть модальное окно</button>
+  <dialog class="dialog" id="some-modal-id">
+    <h1 class="dialog__title">Я модальное окно</h1>
+    <p>Весь остальной контент на странице недоступен, когда я открыто.</p>
+    <form method="dialog">
+      <button class="button button--dark">Понял(а)-принял(а) 👍</button>
```

Comment:
```suggestion
      <button class="button button--dark">Ясно-понятно 👍</button>
```

Можно избавиться от окончаний в скобках, используя более гендерно-нейтральную, но такую же мемную фразочку =)